src/main/java/frc/robot/commands/auto/DriveSegmentBaseCommand.java

```diff
@@ -149,4 +149,6 @@ private static Rotation2d getTrajectoryRotation(List<Translation2d> waypoints) {
     protected static final Pose2d WAYPOINT_Q = new Pose2d(0, 0, Rotation2d.fromDegrees(-88.5));
     protected static final Pose2d WAYPOINT_R = new Pose2d(0, 2.25, Rotation2d.fromDegrees(-90));
     protected static final Pose2d WAYPOINT_S = new Pose2d(6.896412, 3.678778, Rotation2d.fromDegrees(-123));
+    protected static final Pose2d WAYPOINT_T = new Pose2d(1.51203 - 0.2 + .3, 6.7617 + 0.5, Rotation2d.fromDegrees(-45)); //back up for Adit
```

Comment:
(positive up, positive left, positive ccw) looking from middle of right wall to hub

to move back from E, add x, subtract y

- [ ] add -0.5 instead of +0.5